src/Umbraco.Tests/Models/DictionaryItemTests.cs

```diff
@@ -23,12 +23,10 @@ public void Can_Deep_Clone()
                 UpdateDate = DateTime.Now,
                 Translations = new[]
                 {
-                    new DictionaryTranslation(new Language("en-AU")
+                    new DictionaryTranslation(new Language("en-AU", "English (Australia)")
                     {
                         CreateDate = DateTime.Now,
-                        CultureName = "en",
                         Id = 11,
-                        IsoCode = "AU",
```

Comment:
This overwrites the ISO code set in the constructor with an invalid code, so I've removed it (same with US below).